### DISCUSSION TITLE
Leave first option selected on picker mode

### DISCUSSION THREAD

**Discussion by italomaia** [helix-editor/helix#9680] [General]
When searching for a option in picker mode ":", after some text is given, the first option of whichever list of options is shown should always be selected. In my use, the first option is the most likely to be chosen but **current behavior** forces me to press "tab" at least once to pick it.

**Comment by pascalkuthe** [comment]
that is not the file picker those are autocompletions. Immidietly selecting the first option does not make sense for autocompletions

**Reply by italomaia** [reply]
Good to know. Sometimes I don't know what minor mode I'm looking at. New to Helix and all. 

Well, I guess selecting something is more productive than selecting nothing given there is a chance something from the suggestion is what the user want, while for sure, nothing is not really what is desired, and, if nothing is acceptable, the user can always Esc-away from the auto-completion, right?
